pedant/decorators.py

```diff
@@ -206,8 +235,13 @@ def log_render(*args, **kwargs):
         return ''
 
     patch_base = patch.object(VariableNode, 'render', log_render)
-    patch_debug = patch.object(DebugVariableNode, 'render', log_debug_render)
-    return lambda f: patch_base(patch_debug(f))
+    patch_all = patch_base
+    if django.VERSION[:3] < (1, 9):
```

Comment:
Nit. Tuples are ordered lexicographically starting from the zeroth index, so I think `if django.VERSION < (1, 9):` is correct and more concise.